e2e/cypress/integration/auth_sso/authentication_4_spec.js

```diff
@@ -277,10 +277,10 @@ describe('Authentication', () => {
         cy.findByText('Members').click();
 
         // # Input email, select member
-        cy.findByText('Add members or email addresses').type('HosseinTheBestProgrammer@Mattermost.com{enter}{enter}');
+        cy.findByText('Add members or email addresses').type('HosseinTheBestProgrammer@Mattermost.com{downarrow}{downarrow}{enter}');
 
         // # Click invite members button
-        cy.findByText('Invite Members').click();
+        cy.findByRole('button', {name: /invite members/i}).click({force: true});
```

Comment:
I'd suggest to make it case sensitive since it's how the user sees it. I think we can use insensitive for case like aria-label and other similar which is not visible to actual user.  
```suggestion
        cy.findByRole('button', {name: 'Invite Members'}).click({force: true});
```